### ISSUE TITLE
comma in Business Group name causes column shift in csv

### ISSUE THREAD

**Issue by arv109** [mulesoft-catalyst/metrics-accelerator#90]
if the business group has a \ in it (for example "MuleSoft \ CS" , the csv output shows "CS" in the date column, and all other data is shifted right.  the column titles remain the same.

email me for an example ! avijayamohan@mulesoft.com

**Comment by olamiral-mulesoft** [comment]
Could you please provide the steps to reproduce? I tried to create a BG with a backslash in its name, but Anypoint Platform does not allow it. Thanks.

**Comment by arv109** [comment]
hi! can you slack or email me and I will send you an example? it is customer data so I don't want to share on github

**Comment by olamiral-mulesoft** [comment]
Issue confirmed: when the BG contains a comma, the CSV is generated as follow:
```
name,value,businessGroup,date,group,environmentType
Users Total,2,Teste\, BG,2021-03-24T10:26:28.03-03:00,Access Management,NA
Active Users,2,Teste\, BG,2021-03-24T10:26:28.03-03:00,Access Management,NA
Inactive Users,0,Teste\, BG,2021-03-24T10:26:28.03-03:00,Access Management,NA
Active Users Last 60 days,1,Teste\, BG,2021-03-24T10:26:28.03-03:00,Access Management,NA
Active Users Last 30 days,1,Teste\, BG,2021-03-24T10:26:28.03-03:00,Access Management,NA
```
In this case, DW should enclose all fields containing the separator (in this case a comma) in double quotes, when the load strategy is set to CSV (for JSON format, BG names containing commas is not an issue).

**Comment by panizzag** [comment]
Fixed on Release 1.7.0. Closing this issue.
